### ISSUE TITLE
Import admin_level=8

### ISSUE THREAD

**Issue by ialokim** [mapanica/nicaragua-import#1]
In order to enable navitia.io in our `ni` coverage to add [`place`](http://doc.navitia.io/#public-transport-objects) values (so e.g. add "Managua" to all stops inside Managua), it needs administrative boundaries with `admin_level=8`.

Up to now there are none in Nicaragua (see [here](http://overpass-turbo.eu/s/wBW)), but it's pretty the same information provided by [settlements](/mapanica/nicaragua-import/tree/master/settlements).

We should probably give this "little" import a high priority. What do you think @xamanu @AltNico @roirobo @porfiriopaiz ?

**Comment by pantierra** [comment]
Yes, this is great. As a starting point I would like to mention this input:

* Overview of admin_level (particularly I want to refer to the information about Nicaragua): https://wiki.openstreetmap.org/wiki/Tag:boundary=administrative
* We started an import initiative once, not sure about the status though:  https://wiki.openstreetmap.org/wiki/Import/Catalogue/Nicaragua/AdminBoundaries

**Comment by ialokim** [comment]
I've been looking into the `city_polygons_wgs84.shp` dataset which should contain the administrative boundaries for `admin_level=8` and noticed that the data seems quite outdated as nearly all of the cities already seem to be bigger than the shapefile states. See the two examples for Estelí and Managua:

![osm-import1](https://user-images.githubusercontent.com/18088991/52297165-647d5980-2980-11e9-8720-4d3e334f6915.png)
![osm-import2](https://user-images.githubusercontent.com/18088991/52297302-bb832e80-2980-11e9-905a-8418d08d4851.png)

So I fear at least the information provided by this dataset is not very useful (anymore) for an import into OSM.

@xamanu How did you get this data? Could they have newer versions which they could provide us too?

**Comment by ialokim** [comment]
They even seem to provide the contours the [cities had back in 1988](https://www.ineter.gob.ni/mapa/50k_1988/Indice%20mapas50k.html), see for example [Estelí](https://www.ineter.gob.ni/mapa/50k_1988/50K/2955-3.jpg) as provided by INETER.

So pretty sure this data will not be helpful for inclusion in OSM, but I am wondering if it is a good idea to just manually add city contours following the current extend (and not backed up by any official data source)?

**Comment by AltNico** [comment]
@ialokim 
> I am wondering if it is a good idea to just manually add city contours following the current extend (and not backed up by any official data source)?

:+1: for this. I think it's better to have at least some data than to have no data.

**Comment by roirobo** [comment]
> So pretty sure this data will not be helpful for inclusion in OSM, but I am wondering if it is a good idea to just manually add city contours following the current extend (and not backed up by any official data source)?

The problem is that neither the current extend is updated. But is the closest we can get so far.
